internal/cmd/db_list.go

```diff
@@ -49,35 +43,10 @@ var listCmd = &cobra.Command{
 			if err != nil {
 				return err
 			}
-
-			inspectRet := InspectInfo{}
-			ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
-			defer cancel()
-			g, ctx := errgroup.WithContext(ctx)
-			results := make(chan *InspectInfo, len(instances))
-			for _, instance := range instances {
-				loopInstance := instance
-				g.Go(func() error {
-					url := getInstanceHttpUrl(settings, &database, &loopInstance)
-					ret, err := inspect(ctx, url, token, loopInstance.Region, verboseFlag)
-					if err != nil {
-						return err
-					}
-					results <- ret
-					return nil
-				})
-			}
-			if err := g.Wait(); err != nil {
-				if errors.Is(err, context.DeadlineExceeded) {
-					return fmt.Errorf("timeout while inspecting database. It's possible that this database is too old and does not support inspecting or one of the instances is not reachable")
-				}
+			inspectRet, err := CalculateInstancesUsedSize(instances, settings, database, token)
+			if err != nil {
 				return err
 			}
```

Comment:
It's better not to fail on error and just show a message that fetching the size wasn't possible.
It works the same way for fetching instance version in db show.
It's better to show what we can and inform that some data was not able to be fetched.